src/javascript/app_2/App/Components/Elements/PositionsDrawer/positions_drawer.jsx

```diff
@@ -0,0 +1,59 @@
+import classNames                     from 'classnames';
+import PropTypes                      from 'prop-types';
+import React                          from 'react';
+import { Scrollbars }                 from 'tt-react-custom-scrollbars';
+import { localize }                   from '_common/localize';
+import { IconMinus }                  from 'Assets/Common';
+import { connect }                    from 'Stores/connect';
+
+class PositionsDrawer extends React.Component {
+    componentDidMount()    { this.props.onMount(); }
+    componentWillUnmount() { this.props.onUnmount(); }
+
+    render() {
+        const {
+            is_positions_drawer_on,
+            toggleDrawer,
+        } = this.props;
+
+        return (
+            <div className={classNames('positions-drawer', { 'positions-drawer--open': is_positions_drawer_on })}>
+                <div className='positions-drawer__header'>
+                    <span className='positions-drawer__icon-main ic-positions' />
+                    <span className='positions-drawer__title'>{localize('Positions')}</span>
+                    <div
+                        className='positions-drawer__icon-close'
+                        onClick={toggleDrawer}
+                    >
+                        <IconMinus />
+                    </div>
+                </div>
+                <div className='positions-drawer__body'>
+                    <Scrollbars
+                        style={{ width: '100%', height: 'calc(100vh - 35px)' }}
+                        autoHide
+                    >
+                        No Running Contracts
```

Comment:
does this need localization?

Reply:
I think this will be changed in future because content is not yet ready, but I localize it in case it stays